src/SignalRServiceExtension/SignalRMultiConnectionInfoAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Description;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    /// <summary>
+    /// An input binding attribute to bind a list of <see cref="EndpointConnectionInfo"/> to the function parameter.
+    /// </summary>
+    /// <remarks>Designed for function languages except C# to customize negotiation routing.</remarks>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    [Binding]
+    public class SignalRConnectionInfoListAttribute : Attribute
```

Comment:
what's the difference between this one and the `SignalRConnectionInfoAttribute`, they looks pretty similar

Reply:
`SignalRConnectionInfoListAttribute` binds multiple connection info objects of all endpoints to the parameter, `SignalRConnectionInfoAttribute`  binds one connection info objects for a endpoint selected by endpoint router. This two bindings just have same input.